File: golang/fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 0; i <= 100; i++ {
		if (i % 15) == 0 {
			fmt.Printf("%d: FizzBuzz\n", i)
		} else if (i % 3) == 0 {
			fmt.Printf("%d: Fizz\n", i)
		} else if (i % 5) == 0 {
			fmt.Printf("%d: Buzz\n", i)
		} else {
			fmt.Println(i)
		}
	}
}

File: java/FizzBuzz.java
import java.util.stream.IntStream;

public class FizzBuzz {
    public static void main(String[] args) {
        IntStream.range(0, 101).forEachOrdered(n -> {
            if (n % 3 == 0 && n % 5 == 0) {
                System.out.printf("%s: FizzBuzz\n", n);
            } else if (n % 3 == 0) {
                System.out.printf("%s: Fizz\n", n);
            } else if (n % 5 == 0) {
                System.out.printf("%s: Buzz\n", n);
            } else {
                System.out.println(n);
            }
        });
    }
}
